api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -594,6 +594,21 @@ message Features {
   bool IsUsageBased = 17 [(gogoproto.jsontag) = "is_usage_based"];
   // Assist enables the Assistant feature
   bool Assist = 18 [(gogoproto.jsontag) = "assist"];
+  // DeviceTrust holds its namesake feature settings.
+  DeviceTrustFeature DeviceTrust = 19 [(gogoproto.jsontag) = "device_trust,omitempty"];
+}
+
+// DeviceTrustFeature holds the Device Trust feature general and usage-based
+// settings.
+// Requires Teleport Enterprise.
+message DeviceTrustFeature {
+  // True if the Device Trust feature is enabled.
+  bool enabled = 1 [(gogoproto.jsontag) = "enabled,omitempty"];
+  // Usage-based limit for the number of registered/enrolled devices, at the
+  // implementation's discretion.
+  // Meant for usage-based accounts, like Teleport Team. Has no effect if
+  // [Features.IsUsageBased] is `false`.
+  int32 devices_usage_limit = 2 [(gogoproto.jsontag) = "devices_usage_limit,omitempty"];
```

Comment:
Does this  variable name need to be written in `snake_case`  style ? I see mixed convention in this PR in scope of the same file: 
```
DeviceTrustFeature DeviceTrust = 19
int32 devices_usage_limit = 2
```